src/github/amexboy/project_gen.clj

```diff
@@ -0,0 +1,129 @@
+(ns github.amexboy.project-gen
+  (:require [clojure.string :as st]
```

Comment:
`clojure.string` is _usually_ import as `str`, it won't interfere with the `str` function.  Just a suggestion tho, not a requirement.